ui/apps/platform/eslint-plugins/pluginGeneric.js

```diff
@@ -116,6 +116,45 @@ const rules = {
     // ESLint naming convention for negative rules.
     // If your rule only disallows something, prefix it with no.
     // However, we can write forbid instead of disallow as the verb in description and message.
+
+    'no-anchor-href-docs-string': {
+        // Full path string lacks what getVersionedDocs function provides:
+        // Include version number so doc page corresponds to product version.
+        // Encapsulate openshift versus redhat in origin of path.
+        meta: {
+            type: 'problem',
+            docs: {
+                description:
+                    'Replace string with getVersionedDocs function call in href prop of anchor element for product docs',
+            },
+            schema: [],
+        },
+        create(context) {
+            const isDocs = (href) =>
+                href.startsWith('https://docs.openshift.com/acs/') ||
+                href.startsWith('https://docs.redhat.com/acs/');
```

Comment:
@kcarmichael08 These patterns match unwanted full paths to replace with helper function calls.
* Do you know if `acs` still follows the origin as first-level folder in the new site structure?
* Do you know if version like `4.5` still follows as second-level folder in the path?

Reply:
> @kcarmichael08 These patterns match unwanted full paths to replace with helper function calls.
> 
> * Do you know if `acs` still follows the origin as first-level folder in the new site structure?

> * Do you know if version like `4.5` still follows as second-level folder in the path?

It's a bit different. Here's an example:

docs.openshift.com:

https://docs.openshift.com/acs/4.5/configuration/add-custom-certificates.html

docs.redhat.com:

https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_security_for_kubernetes/4.5/html/configuring/add-custom-cert

I am not sure if it is OK to just refer to the English docs, and how the language affects the part after `docs.redhat.com`. (I can't figure out how to change the language.) Then the rest of it is a little different too.

Reply:
Super, thank you!
* Because the lint rule prevents copy-paste from browser address, `en` might be good enough.
* We will review the subject again with y’all when it is time to update `getVersionedDocs` function.